Adding some corrections to DBaaS (#1661)
CSCfi/csc-user-guide: docs/cloud/dbaas/web-interface.md

On the next page `Database access`:

7. `Allowed CIDRs` - Here you want to add your allowed IP-addresses in the format `$IP/32` if you want to allow multiple IP-addresses. You need to separate them by a comma `,`.

On the third page `Intialize Databases:

8. In the `Initial Databases` field you can write what databases should be initialized. You can add additional database after the instance have started.
9. `Initial Admin User` adding first user that you want to use to connect to the database. You can add more user accounts after the database instance have started.
10. `Password` for the you first user. Please make sure that this password is not used anywhere else.
11. `Allowed Host (optional)` , this feature is not supported at the moment but in the future it will possible to limit users access based on IP.

On the fourth page `Advanced`:

12. You don't need to do anything but it is possible to launch a new database from a backup. If you choose to launch a database from a backup you don't need to specify `Initial Admin User`, `Passowrd`, or `Initial Databases`
13. Now you can press `Launch`

> 14. Once the database instance has launched. You can press on the name of the database instance and manage additional Users, databases backups a.s.o. but it might take a couple minutes before the instance is up and running.

Reviewer: ```suggestion
14. Once the database instance has launched, you can click the name of the database instance and manage additional items, such as users, backups etc. But it might take a couple minutes before the instance is up and running.
```

Author: Updated as suggested. Thanks for the suggestions.
